### PR TITLE
fix(duckduckgo): misc

### PR DESCRIPTION
**Pull request by egirlcatnip**
## 🔧 What does this fix? 🔧
- Fixed Text overflow fade in scrollable toolbox
- Themed Duckduckgo open in application button in header of the page
- Themed Mobile footer and footer separator 
- Tweaked settings panel's colors
- Themed Mobile More results button
- Edited On/Off switches for clarity

note: still needs some aditional finishing touches, but this should be the majority of the fixes I could find

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Comment by GenShibe** [comment]
image

personally, i feel that there's not enough contrast for the on/off toggle, but otherwise it looks fine to me

paging @uncenter and @isabelroses to get their expert opinions on the matter

**Comment by uncenter** [comment]
> image
> 
> personally, i feel that there's not enough contrast for the on/off toggle, and that the button itself should be something like overlay0 to make it more visible against the background but otherwise it looks fine to me
> 
> paging @uncenter and @isabelroses to get their expert opinions on the matter

From that screenshot I would think that that toggle is disabled. The thumb definitely needs a brighter color to appear active but off.

**Comment by egirlcatnip** [comment]
no problem, this can be adjusted to eg.  ```@text```

**Comment by egirlcatnip** [comment]
I can make the change later in the day, or someone else can.

File: styles/duckduckgo/catppuccin.user.css [hunk 1]
```diff
@@ -770,11 +792,11 @@
...
       color: @accent-color !important;
     }
-    .switch {
+     .switch, .frm__switch__label {
       background-color: @mantle !important;
     }
-    .switch__knob {
-      background: @text !important;
+    .switch__knob, .frm__switch__label::after {
+      background: @base !important;
```
Comment (GenShibe) [inline styles/duckduckgo/catppuccin.user.css:799]: This should not be base as there is a lack of contrast.

**Review by GenShibe — CHANGES_REQUESTED** [review]

**Comment by egirlcatnip** [comment]
I'm trying to set something up

image
image

image
image

image
image

image
image


Your thoughts?


```css
.switch__knob {
  background-color: @crust;
}

.switch:not(.is-on) > .switch__knob {
  background-color: @overlay2;
}



```

**Comment by egirlcatnip** [comment]
Alright, pushed a commit for the switches, should be fine now!

**Pull request merged**